dbt_subprojects/daily_spellbook/models/xrpl/transfers/tokens_xrpl_amm_deposits.sql

```diff
@@ -0,0 +1,303 @@
+{{
+    config(
+        schema = 'tokens_xrpl_amm_deposits',
+        alias = 'transfers',
```

Comment:
```suggestion
        schema = 'tokens_xrpl',
        alias = 'amm_deposits_transfers',
```

Reply:
for all of these individual transfer models, lets use this naming standard.
keeps schemas clean, puts type in table name.

Reply:
plz apply to all models.